### PR TITLE
mpu6050: Add fixes to kernel module and program to check performance

### PR DESCRIPTION
**Pull request by BekirovBekir**
1) Fixes to driver of mpu6040 have been added. 
2) Optimization for mpu6050 driver has been implemented.
3) User space program to check performance was implemented.

### CODE REVIEW

File: mpu6050/perfomance_checking_program/main.h [hunk 1]
```diff
@@ -0,0 +1,34 @@
+/*
+ * main.h
+ *
+ *  Created on: 17 нояб. 2019 г.
```
Comment (yekovalyov) [inline mpu6050/perfomance_checking_program/main.h]: It is better to translate date to English and add author's email.

**Comment by yekovalyov** [comment]
The task has been done well. Please check developed code using checkpatch.pl. For clarifying the changes in code it is better to add updates in the same file with additional commit.

File: mpu6050/perfomance_checking_program/main.h (hunk 1 above)
Comment (BekirovBekir) [reply mpu6050/perfomance_checking_program/main.h]: I've fixed this error and some mistakes in a spelling.

File: mpu6050/performance_checking_program/main.c [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+			time_ns = time_interval(&ptr_data->time_end,
+					&ptr_data->time_start);
+			printf("%s\r\x1b[48C%llu\n", sysfs_file_name, time_ns);
+			close(fd);
+			return 0;
+		} else {
+			printf("Error while reading file!\n");
+			close(fd);
+			return 1;
+		}
```
Comment (yekovalyov) [inline mpu6050/performance_checking_program/main.c]: Is this indentation allow by checkpatch.pl?

**Comment by yekovalyov** [comment]
How many values per second is it possible to get using implemented test program?

File: mpu6050/performance_checking_program/main.c (hunk 2 above)
Comment (BekirovBekir) [reply mpu6050/performance_checking_program/main.c]: It seems yes. The checkpatch has showed neither warning or error in that place.

**Pull request merged**